app/main/views/users.py

```diff
@@ -63,6 +63,12 @@ def list_users():
             links={}
         )
 
+    only_buyers = request.args.get('only_buyers')
```

Comment:
Could it be worth making this functionality more general so you can effectively filter by any user role rather than just for the buyer role? (feel free to say no)